src/tumpa/StartView.qml

```diff
@@ -8,68 +9,58 @@ Rectangle {
     color: "white"
     signal genkeyclicked
 
-    Image {
-        id: bigKey
-        source: "images/big_key.svg"
-        anchors.topMargin: 140
-        anchors.top: root.top
-        anchors.left: root.left
-        anchors.leftMargin: 326
-    }
+    ColumnLayout {
+        anchors.centerIn: parent
+        spacing: 20
 
-    Text {
-        id: nokeyTxt
-        text: qsTr("No keys added yet")
-        font.pixelSize: 20
-        font.weight: 700
-        anchors.top: bigKey.bottom
-        anchors.topMargin: 12
-        anchors.left: root.left
-        anchors.leftMargin: 283
-        color: "black"
-    }
+        Image {
+            id: bigKey
+            Layout.alignment: Qt.AlignHCenter
+            source: "images/big_key.svg"
+        }
 
-    Text {
-        id: bigMsgTxt
-        text: qsTr("You can import an existing key or generate a new one")
-        font.pixelSize: 14
-        font.weight: 400
-        anchors.top: nokeyTxt.bottom
-        anchors.topMargin: 20
-        anchors.left: root.left
-        anchors.leftMargin: 180
-        anchors.bottomMargin: 16
-    }
+        Text {
+            id: nokeyTxt
+            Layout.alignment: Qt.AlignHCenter
+            text: qsTr("No keys added yet")
+            font.pixelSize: 20
+            font.weight: 700
+            color: "black"
+        }
+
+        Text {
+            id: bigMsgTxt
+            Layout.alignment: Qt.AlignHCenter
+            text: qsTr("You can import an existing key or generate a new one")
+            font.pixelSize: 14
+            font.weight: 400
+        }
 
-    Row {
-        id: buttonRow
-        anchors.top: bigMsgTxt.bottom
-        anchors.topMargin: 20
-        anchors.left: root.left
-        anchors.leftMargin: 178
-        spacing: 10
+        RowLayout {
```

Comment:
Just to understand better for future coding, any reason for prefering RowLayout over Row? I think we have usually kind of tried to avoid using Layouts (not for any specific reason, just to probably reduce the number of things we need to import).